### ISSUE TITLE
sqlqueryreceiver lacks integration tests for mssql

### ISSUE THREAD

**Issue by cwegener** [open-telemetry/opentelemetry-collector-contrib#29695]
### Component(s)

receiver/sqlquery

### Describe the issue you're reporting

When submitting #29694 I noticed that only Oracle, Postgres and MySQL are currently covered with integration tests.

Adding MSSQL as another popular RDBMS to the existing integration tests should be considered

**Comment by crobert-1** [comment]
Good idea, and I believe [this would be the best place](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/sqlqueryreceiver/integration_test.go) to put it.

**Comment by bradoyler** [comment]
Does anyone have a working datasource string for this?
